assets/collage.css

```diff
@@ -474,8 +474,22 @@
 .collage-card:not(.collage-product, .collage-collection) {
   border: var(--media-border-width) solid rgba(var(--color-foreground), var(--media-border-opacity));
   border-radius: var(--media-radius);
-  box-shadow: var(--media-shadow-horizontal-offset)
-    var(--media-shadow-vertical-offset)
-    var(--media-shadow-blur-radius)
-    rgba(var(--color-foreground), var(--media-shadow-opacity));
+  background: rgb(var(--color-background));
+}
+
+.collage-card:not(.collage-product, .collage-collection):after {
+  border-radius: var(--media-radius);
+  box-shadow: var(--media-shadow-horizontal-offset) var(--media-shadow-vertical-offset) var(--media-shadow-blur-radius) rgba(var(--color-foreground), var(--media-shadow-opacity));
+  content: '';
+  position: absolute;
+  width: calc(var(--media-border-width) * 2 + 100%);
+  height: calc(var(--media-border-width) * 2 + 100%);
+  top: calc(var(--media-border-width) * -1);
+  left: calc(var(--media-border-width) * -1);
+  z-index: -1;
+}
```

Comment:
I'm wondering if we should treat this as a card instead 🤔  So it matches the styles from the product or collection block.
Otherwise to tweak this section, you would have to touch the `cards` settings as well as the `media` ones. Curious to hear your thoughts on this.

Reply:
We will once we tackle this issue https://github.com/Shopify/dawn/issues/1015